docs/Documentation/Scripting/Building-Blocks/Conditions-List.md

```diff
@@ -489,16 +489,16 @@ This condition is met if the block at specified location matches the given mater
 
 There must be specific (Minecraft) time on the world for this condition to return true.
 
-| Parameter  | Syntax     | Default Value          | Explanation                                                        |
-|------------|------------|------------------------|--------------------------------------------------------------------|
-| _Variable_ | time span  | :octicons-x-circle-16: | Two hour numbers seperated by dash in the 24-hour format (0 - 24). |
-| _world_    | world:name | player location        | The world to check for the time. Can be a variable.                |
+| Parameter  | Syntax     | Default Value          | Explanation                                                                        |
+|------------|------------|------------------------|------------------------------------------------------------------------------------|
+| _Variable_ | time span  | :octicons-x-circle-16: | Two time points (hh or hh:mm) seperated by dash in the 24-hour format (0 - 23:59). |
```

Comment:
Is it actually, that as we have minutes now, it is from 0-59? what happens if 24 is still used? does it break then? also this would be more complicated to write as the previous format.

Reply:
Added the possibility of using 24. It's still possible to only write the hours, so no additional complication, only more possibilities.